.github/ISSUE_TEMPLATE/feature.md

```
@@ -0,0 +1,44 @@
+---
+name: Feature Template
+about: Create a feature request
+title: ""
+labels: "feature, status:new"
+assignees: ""
+---
+
+<!--- Please DO NOT remove the automatically added 'new issue' label -->
+<!--- Provide a general summary of the issue in the Title above -->
+
+### Is your feature related to a problem?
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)